fmriprep/interfaces/reports.py

```diff
@@ -16,7 +16,7 @@
     traits, TraitedSpec, BaseInterfaceInputSpec,
     File, Directory, InputMultiObject, Str, isdefined,
     SimpleInterface)
-from nipype.interfaces import freesurfer as fs
+from smriprep.interfaces.freesurfer import ReconAll
```

Comment:
I'm not actually sure we need that. All that patch allows us to do is to specify a collection of `steps` instead of a `directive`. We are testing the directive `all` below.

Reply:
i added that import to keep the `recon-all` calls consistent - lmk if you feel strongly to change that

Reply:
I don't feel strongly.